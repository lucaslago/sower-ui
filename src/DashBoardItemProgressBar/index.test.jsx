import DashboardItemProgressBar from './index';

describe('<DashboardItemProgressBar>', () => {
  const simulationService = { status: () => {} };
  const simulationStatus = { status: 'inactive' };
  it('should render correctly', () => {
    const wrapper = shallow(<DashboardItemProgressBar
      trackerId={'123'}
      authToken={'Bearer 12312312'}
      simulationService={simulationService}
      simulationStatus={simulationStatus}
    />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
