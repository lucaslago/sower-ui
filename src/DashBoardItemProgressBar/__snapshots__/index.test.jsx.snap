exports[`<DashboardItemProgressBar> should render correctly 1`] = `
<div
  className="dashboard-item-progress-bar">
  <Subheader
    inset={false}>
    /
  </Subheader>
  <LinearProgress
    max={100}
    min={0}
    mode="determinate"
    value={0} />
</div>
`;
